client/src/styles/legacy/GridField.scss

```diff
@@ -98,6 +98,11 @@
       margin-right: 0;
       display: inline-block;
     }
+
+    .action-menu__toggle {
+      padding-top: 0.25rem;
+      padding-bottom: 0.25rem;
```

Comment:
I think the linter will require `.25rem`

Reply:
We don't actually enforce a rule either way and from a quick search we seem to prefer a leading zero